app/controllers/keywords_controller.rb

```diff
@@ -1,5 +1,25 @@
 # frozen_string_literal: true
 
+require 'csv'
+
 class KeywordsController < ApplicationController
   def index; end
+
+  def create
+    keywords = load_keywords
+    Rails.logger.debug(keywords)
```

Comment:
Can we remove the logger here? 🤔

Reply:
Fixed in [`ada71a1` (#38)](https://github.com/suho/google-search-ruby/pull/38/commits/ada71a118f36016a575b128ffac04d1646c83b98)